src/table/components/head/MenuList/MenuGroup.tsx

```diff
@@ -27,10 +27,66 @@ export const interceptClickOnMenuItems = (menuGroups: MenuItemGroup[], cache: Su
 type SubMenusOpenStatusCache = Record<string, React.Dispatch<React.SetStateAction<boolean>>>;
 let subMenusOpenStatusCache: SubMenusOpenStatusCache = {};
 
-const MenuGroupItems = ({ id, onClick, itemTitle, icon, enabled, subMenus }: HeadCellMenuItem) => {
+const MenuGroupItems = ({ autoFocus, id, onClick, itemTitle, icon, enabled, subMenus }: HeadCellMenuItem) => {
   const [openMenu, setOpenMenu] = useState(false);
   const anchorRef = useRef<HTMLDivElement | null>(null);
 
+  const moveToNextFocusItem = (currentFocus: Element) => {
+    let nextItem = currentFocus.nextElementSibling as HTMLDivElement;
+    if (!nextItem) {
+      nextItem = currentFocus.parentElement?.children[0] as HTMLDivElement;
```

Comment:
You can return the element instead of reassigning the variable

Reply:
Updated